proto/pdpb.proto

```diff
@@ -389,6 +393,22 @@ message GetMembersResponse {
     map<string, Member> tso_allocator_leaders = 5;
 }
 
+message GetClusterInfoRequest {
+    ResponseHeader header = 1;
+}
+
+enum ServiceMode {
+    UNKNOWN_SVC_MODE = 0;
```

Comment:
Do we need unknown?

Reply:
Yes, it needs to an initial value at startup.
1. If I initialize it with PD module, it actually introduce more complexity at startup.
2. We might need to set UNKNOWN_SVC_MODE by server side if the service is in switching, e.g., half servers running in pd mode; and half servers running in api mode, so we might need one more state.